package.json

```diff
@@ -33,6 +33,7 @@
     "linefeed"
   ],
   "devDependencies": {
-    "mocha": "*"
+    "ava": "^0.14.0",
```

Comment:
@jamestalmage See: https://github.com/sindresorhus/split-lines/pull/1#discussion_r64174682 Yes, there is a danger, and I wouldn't recommend it for others, but I do it in my own projects for a good reason.